ci/playbooks/deploy_watcher_service.yaml

```diff
@@ -35,9 +35,9 @@
         script: make watcher
         extra_args:
           CATALOG_IMAGE: "{{ watcher_catalog_image | default('quay.io/openstack-k8s-operators/watcher-operator-index:latest') }}"
-          RELATED_IMAGE_WATCHER_API_IMAGE_URL_DEFAULT: "{{ _registry_url }}/openstack-watcher-api:{{ _tag }}"
-          RELATED_IMAGE_WATCHER_DECISION_ENGINE_IMAGE_URL_DEFAULT: "{{ _registry_url }}/openstack-watcher-decision-engine:{{ _tag }}"
-          RELATED_IMAGE_WATCHER_APPLIER_IMAGE_URL_DEFAULT: "{{ _registry_url }}/openstack-watcher-applier:{{ _tag }}"
+          WATCHER_API_IMAGE: "{{ _registry_url }}/openstack-watcher-api:{{ _tag }}"
```

Comment:
Doesn't look correct too:
https://github.com/openstack-k8s-operators/watcher-operator/blob/de3399195bb8d03c77be796650cd843d153c145d/Makefile#L386-L389
I think that we should also renamed the var in Makefile, since
`WATCHER_*_IMAGE_URL_DEFAULT_MASTER`
is not a good var name

Reply:
Done!